timber/src/main/java/timber/log/Timber.java

```diff
@@ -116,6 +117,30 @@ public static void plant(Tree tree) {
     }
   }
 
+  /** Add a new logging tree. */
+  public static void plant(Tree... trees) {
+    if (trees == null) {
+      throw new NullPointerException("trees == null");
+    }
+    if (trees.length == 0) {
+      return;
+    }
+    //noinspection ForLoopReplaceableByForEach
+    for (int i = 0; i < trees.length; i++) {
```

Comment:
Yeah. That'd be great

On Wed, Jul 6, 2016 at 2:52 PM Prateek Srivastava <notifications@github.com>
wrote:

> In timber/src/main/java/timber/log/Timber.java
> <https://github.com/JakeWharton/timber/pull/150#discussion_r69789751>:
>
> > @@ -116,6 +117,30 @@ public static void plant(Tree tree) {
> >      }
> >    }
> >
> > +  /** Add a new logging tree. */
> > +  public static void plant(Tree... trees) {
> > +    if (trees == null) {
> > +      throw new NullPointerException("trees == null");
> > +    }
> > +    if (trees.length == 0) {
> > +      return;
> > +    }
> > +    //noinspection ForLoopReplaceableByForEach
> > +    for (int i = 0; i < trees.length; i++) {
>
> Cool updated, might update the tree of souls iteration separately.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/JakeWharton/timber/pull/150/files/24924a2d83b48dde6d6bf9ada8f3c5de091107d2#r69789751>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEEEeYhQoUjeoJ3cIHu0JBECScQk6DLks5qS_lcgaJpZM4JFmpd>
> .
>